modules/replicator/src/main/java/org/apache/ignite/internal/replicator/message/ReplicaRequest.java

```diff
@@ -28,5 +29,6 @@ public interface ReplicaRequest extends NetworkMessage {
      *
      * @return Replication group id.
      */
-    String groupId();
+    @Marshallable
+    TablePartitionId groupId();
```

Comment:
Earlier we've discussed that ReplicaRequest is a general entity that also will be used within meta storage etc. Thus it's not valid to use stuff like Table in int.

Reply:
ReplicationGroupId that is replicator module should no nothing about the table. What's the problem in implementing proposed solution with an interface and TablePartitionId implementation.

>I think, it is enough this time, we will have added something else when will use replication for metastorage.
Why would we implement incorrect solution, within the context of plans of moving meta storage towards replicas.